apps/dashboard/app/helpers/batch_connect/session_contexts_helper.rb

```diff
@@ -15,6 +15,8 @@ def create_widget(form, attrib, format: nil)
       form.form_group attrib.id, help: field_options[:help] do
         form.check_box attrib.id, all_options, attrib.checked_value, attrib.unchecked_value
       end
+    when "radio", "radio_button"
+      form.collection_radio_buttons attrib.id,   attrib.select_choices, :second, :first, checked: [attrib.value] + Array.wrap(attrib.field_options[:checked])
```

Comment:
What is the behavior if both a `value` and `checked` are set?

Reply:
Good call. I was in the process of explaining why this worked when I found a bug. Now the "checked" value will use `attrib.value` unless `attrib.value.blank?`, then the `checked` value is used.